### ISSUE TITLE
nonjailbroken ios

### ISSUE THREAD

**Issue by zakusa** [WilliamLCobb/GC4iOS#16]
how do you get gc4ios on nonjailbroken ipad i have ipad air 2 ios 9.3.2

**Comment by MaquinaDelMisterio** [comment]
Download the github files, and open xcode. Project.

**Comment by zakusa** [comment]
i dont get it at all

**Comment by zakusa** [comment]
where do i get xcode and how do i open it and thanks for replying fast

**Comment by MaquinaDelMisterio** [comment]
Go to the MacStore and download xcode, then download this file https://github.com/WilliamLCobb/GC4iOS/archive/master.zip open it and search for xcode project file, its a blue print icon, just double click it, plug in your device and press play button.

**Comment by zakusa** [comment]
i dont have mac i cant get xcode is it possible to do it on windows

**Comment by MaquinaDelMisterio** [comment]
Virtual machine. 😉

**Comment by zakusa** [comment]
that suxs but ok

**Comment by MaquinaDelMisterio** [comment]
Thats the only way if you are using windows.

**Comment by zakusa** [comment]
i change my mind im actually going to do it on my old laptop using virtual machine because it look sooo cooool

**Comment by zakusa** [comment]
i have spent 2 hours trying to get dmg to work in virtualbox do you know where i can get a iso

**Comment by hjstn** [comment]
I don't believe it's legal to get an OS X image on a non-Mac computer, but they're there on torrent sites. AFAIK though you can't run OS X in a VM without the host being a Mac.

**Comment by zakusa** [comment]
help i have my ipad plugged in and when i open the xcode project file i get this window i dont see a play button 

http://prntscr.com/bi18f5

**Comment by peter961785** [comment]
view > show toolbar

**Comment by zakusa** [comment]
ok i didnt really do view > show toolbar but when i opened the project file again heres what i saw ( the link below has the picture ) so i clicked view and i saw "hide toolbar" i didnt play around because i didnt want the screen to change so i just paused vmware workstation

http://prntscr.com/bif715

**Comment by hjstn** [comment]
The triangle on the top left is the play button you're looking for, but you also need to have an Apple Developers account (https://developer.apple.com/register/agree/, and sign the legal document). And then,
1) Preferences -> Accounts -> Click the + and add your Apple ID if you haven't, then click your apple ID under Apple IDs and you should see something like this, with your name instead of mine.
![image](https://cloud.githubusercontent.com/assets/2884155/16185195/b3c67c74-36df-11e6-8a34-1891d02efc18.png)

2) Close Preferences, and like in your screenshot above, you should see a title "Identity", there you should see Team, that should be set to the team in the screenshot above, like so: 
![image](https://cloud.githubusercontent.com/assets/2884155/16185248/04b21d00-36e0-11e6-9f65-b6e87086f854.png)

3) There should be a warning saying "No provisioning certificates found", first change the "Bundle Identifier" above to something like "zakusa.GC4iOS", and then click "Fix Issue" and follow the steps until the warning is gone.

Then, on the top left, next to the square button, there should be something like this:
![image](https://cloud.githubusercontent.com/assets/2884155/16185270/3f5d940c-36e0-11e6-865a-1ca95d87203a.png)
(NOTE: Mine says iNDS because that's what I was using for the example, but yours should say GC4iOS or something along the lines of it)

If it shows a hammer or a blueprint+iPhone/iPad, connect your iDevice, and click the hammer/blueprint and select your device name from there, like you did here: 
![image](https://cloud.githubusercontent.com/assets/2884155/16185285/5c4bf522-36e0-11e6-868c-f5fb0ef20121.png)

Then, just hit the triangle next to the square button and watch it load.

After it is installed, there is another step before you can run it, go to Settings -> General -> Profiles & Device Management on your iPad, there should be your email under "Developer App", click it, then click "Trust [your Apple ID here]", or refer to https://support.apple.com/en-us/HT204460

Then you should be able to load ROMs under Apps in iTunes (make sure to unzip them!) and play!
Another Note: As you're running a jailed version of iOS like me, GC4iOS does not (and cannot at the moment) use JIT, which is more efficient for emulation, so games will most likely be slow to the point that it's unplayable, or crash frequently.

**Comment by zakusa** [comment]
"could not build developer image" but whatever im done i looked up on google and it said i need to upgrade os x which i have a 10.11 and ipad doesnt wrk there i even looked up on google but ill wait for a jailbreak instead

**Comment by DarknesGaming** [comment]
I know it's too late to say this, but VirtualBox is NOT recommended due to its slow speed. I recommend using VMWare, and xBytez's Virtual Disk, which contains Mac OS X El Capitan (10.11).

**Comment by zakusa** [comment]
i did use vmware in the picture above look at it

**Comment by zakusa** [comment]
who else used vmware workstation to install dolphin on ios maybe you can give me video you used and i try it

**Comment by zakusa** [comment]
ok guys i got gc4ios  ya justin i followed your steps and it worked coooool but i cant get roms i tried dropbox, google drive, itunes ( gc4ios doesnt show up in itunes)

**Comment by MaquinaDelMisterio** [comment]
@zakusa Do the following: Go to GC4IOS folder - GC4IOS - Resources - roms - paste it there. Then delete the app from you idevice and load it again with xcode

**Comment by gottagetitguy** [comment]
Please help I'm following these instructions but it keeps failing can someone do the most in depth tutorial they can please I really wanna play my gc games

**Comment by MaquinaDelMisterio** [comment]
Im sorry to tell you that even you are jailbroken emulation doesnt work. I suggest using the Android variant, that one is working.

**Comment by zakusa** [comment]
@gottagetitguy get the gc4ios repo and download gc4ios in it ya games work but it's very slow and sometimes games just crash at anytime I don't know is William Cobb is working in it or he will do it later
